projects/ccf-database/src/lib/hubmap/hubmap-data.ts

```diff
@@ -170,9 +170,9 @@ export class HuBMAPEntity {
     let bmi: number | undefined;
     let ethnicity: string | undefined;
     for (const md of get(this.donor, 'metadata.organ_donor_data', []) as JsonDict[]) {
-      if (md.preferred_term === 'Feminine gender' || md.preferred_term === 'Female') {
+      if (md.preferred_term === 'Feminine sex' || md.preferred_term === 'Female') {
```

Comment:
preferred_term still has to be 'Feminine gender' and 'Masculine gender' as this is how the data comes in.